### DISCUSSION TITLE
Bookmark function

### DISCUSSION THREAD

**Discussion by magdalenakierat** [neuefische/questions#33] [Web] [answered]
Hallo zusammen,
das neue Bookmark Feature funktioniert auf den ersten Blick, aber eine Sache bekomme ich nicht in den Griff:
Wenn das Bookmark bei einer Karte, durch Klick aufs Herz, gesetzt wurde (Herz = gefüllt), behält die Karte diesen Zustand, wenn danach der „Favoriten“ oder der „Alle“ Button geklickt wurde. So soll es auch sein. 
Wenn aber einer der anderen Filter-Buttons (Bewegung, Atem, Augen) geklickt wird, wird der visuelle Bookmark-Zustand wieder auf default gesetzt (Herz = nicht gefüllt), obwohl der eigentliche Bookmark-Zustand anders ist (bookmarked=true). 
Frage: Wie schaffe ich es, dass das Herz immer den richtigen Zustand widerspiegelt, unabhängig davon welcher Filter-Button geklickt wurde?
Hier der Link zum entsprechenden Branch: https://github.com/magdalenakierat/capstone-project/compare/bookmark?expand=1
1000 Dank für jeden Hinweis! 🙏

**Comment by mbosselmann** [comment, marked as answer]
Hallo Magda,
in deiner `Cards.js` übergibst du den key `bookmarked` nicht an deine `Card` Komponente. Wenn du `bookmarked={exercise.bookmarked}` hinzufügst, sollte es funktionieren. :)
Schönes Wochenende!

**Reply by magdalenakierat** [reply]
Danke, liebe Mareike! Jetzt funktioniert es!! 🎉
